org.eclipse.lsp4e.debug/src/org/eclipse/lsp4e/debug/debugmodel/DSPBreakpointManager.java

```diff
@@ -68,7 +68,7 @@ public DSPBreakpointManager(IBreakpointManager platformBreakpointManager, IDebug
 	 *
 	 * @return the completeable future to signify when the breakpoints are all sent.
 	 */
-	public CompletableFuture<Void> initialize() {
+	public CompletableFuture<@Nullable Void> initialize() {
```

Comment:
What is the reason for it? A limitation in the JDT?

Is Void not type used to express that the method does not return a value, that is more or less equivalent to null?

Reply:
Void futures are completed with a null value. Otherwise you cannot complete it.